userStoryInventoryChangHF.md

```diff
@@ -39,41 +39,646 @@ This section shows the areas or sources thus far
+
+## 2R: Delegate to No-Confidence Feature for DReps
+
+**User Story:** As a DRep, I want to be able to delegate using the "delegate to no-confidence" feature ensuring only my own lovelace's voting power is delegated to No-Confidence without affecting voting power delegated to me by others.
+
+### Acceptance Criteria
+
+- Given that I am a DRep.
+- When I delegate using the "delegate to no-confidence" feature.
+- Then only my own lovelace's voting power will be delegated to No-Confidence, excluding any voting power delegated to me by others. And I will be notified that my delegation transaction was sent.
+
+## 2S: Delegate to No-Confidence Feature for Non-DReps
+
+**User Story:** As a non-DRep user, I want to be able to delegate using the "delegate to no-confidence" feature ensuring any voting power I have is delegated to No-Confidence.
+
+### Acceptance Criteria
+
+- Given that I am not a DRep.
+- When I delegate using the "delegate to no-confidence" feature.
+- Then any voting power I have will be delegated to No-Confidence, and I will be notified that my delegation transaction was sent.
+
+## 3A: Confirmation of dRep Registration
+
+**User Story:** As a dRep, I want to be able to see if I am registered so that I can confirm my registration.
+
+### Acceptance Criteria
+
+- Given that I have registered as a dRep.
+- When I look at any page on the app.
+- Then I should see a widget at the top right of each page showing the voting power that has been delegated to me and my dRep ID. On the governance action page, I also have the option to cast a vote.
+
+## 3B: Dedicated Page for Registering as a dRep
+
+**User Story:** As a dRep (someone who wants to be a dRep), I want one page dedicated to registering as a dRep so that I can easily register as a dRep.
+
+### Acceptance Criteria
+
+- Given that I have the app open.
+- When I am seeking to register as a dRep.
+- Then I should be able to access a page dedicated to registering as a dRep.
+
+## 3C: Registering as dRep on Disconnected Wallet State
+
+**User Story:** As a dRep (someone who wants to be a dRep) who has not connected their wallet to the app, I want to be prompted to connect my wallet to the app if I try to register as a dRep so that I can register as a dRep.
+
+### Acceptance Criteria
+
+- Given that I have not logged in to the app.
+- When I visit the dRep registration page.
+- Then I should be prompted to log into the app before I can register. I cannot begin the process until I have logged in.
+
+## 3D: Registering as a dRep on Connected Wallet State
+
+**User Story:** As a dRep (someone who wants to be a dRep) who has connected their wallet to the app, I want to be able to register as a dRep so that I can fulfill the duties of a dRep.
+
+### Acceptance Criteria
+
+- Given that I have logged into the app.
+- When I visit the dRep registration page.
+- Then I should be presented with a registration form consisting of:
+  - An input box for my dRep metadata URL.
```

Comment:
what about the metadata form we have currently?

Reply:
This has been updated with the latest test.